packages/calcite-components/src/components/chip/chip.stories.ts

```diff
@@ -42,6 +42,7 @@ export const simple = (args: ChipStoryArgs): string => html`
       scale="${args.scale}"
       appearance="${args.appearance}"
       kind="${args.kind}"
+      label="My great chip"
```

Comment:
Can you provide a label arg for this story? You'll need to update the [`ChipStoryArg` type](https://github.com/Esri/calcite-design-system/blob/dev/packages/calcite-components/src/components/chip/chip.stories.ts#L10) and add an entry for the [default value](https://github.com/Esri/calcite-design-system/blob/dev/packages/calcite-components/src/components/chip/chip.stories.ts#L14-L19). This can be done as a follow-up to proceed with the merge.

Reply:
Added via https://github.com/Esri/calcite-design-system/pull/10888/commits/d43e01f93822a8a16e6271c6b29d03a8b97e2955, does this look accurate, @jcfranco? 👀